packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -121,6 +119,13 @@ export class InsuredBridgeL2Client {
     });
   }
 
+  async getEventsForBlockSearch(blockSearchConfig: { fromBlock: number; toBlock: number }): Promise<EventData[][]> {
+    return await Promise.all([
+      this.bridgeDepositBox.getPastEvents("FundsDeposited", blockSearchConfig),
+      this.bridgeDepositBox.getPastEvents("WhitelistToken", blockSearchConfig),
+    ]);
+  }
```

Comment:
Two nits:
1. Should the return values be named? Without looking at the implementation, it's hard to tell what return[0] and return[1] are.
2. getEventsForBlockSearch seems like an odd method for a general-purpose client. This combined call makes sense for our relayer but may be overkill for other uses. Thoughts on 1) separating these queries into two methods and 2) naming them more specifically so it's clear what the method will be returning?

Reply:
On point 2 do you mean making one GET method for each event name?

Reply:
Yeah. I know that's kinda annoying since it's more verbose.

We could just do 1 and have the interface more clearly communicate what is being returned.

My main push is that the user should be able to look at the interface (`
  async getEventsForBlockSearch(blockSearchConfig: { fromBlock: number; toBlock: number }): Promise<EventData[][]>`) and know what the method is going to give them and how to use it -- any way we can achieve that SGTM.

Thoughts?

Reply:
No I think you're right about having one method for each event name, I've implemented the change.